internal/cmd/alpha/registry/config/config.go

```diff
@@ -0,0 +1,65 @@
+package config
+
+import (
+	"fmt"
+	"github.com/kyma-project/cli.v3/internal/clierror"
+	"github.com/kyma-project/cli.v3/internal/cmdcommon"
+	"github.com/kyma-project/cli.v3/internal/registry"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+type cfgConfig struct {
+	*cmdcommon.KymaConfig
+	cmdcommon.KubeClientConfig
+
+	dockerconfig bool
+	externalurl  bool
+	output       string
+}
+
+func NewConfigCMD(kymaConfig *cmdcommon.KymaConfig) *cobra.Command {
+	cfg := cfgConfig{
+		KymaConfig:       kymaConfig,
+		KubeClientConfig: cmdcommon.KubeClientConfig{},
+	}
+
+	cmd := &cobra.Command{
+		Use:   "config",
+		Short: "Saves Kyma registry dockerconfig to a file",
+		Long:  "Use this command to save Kyma registry dockerconfig to a file",
+		PreRun: func(_ *cobra.Command, _ []string) {
+			clierror.Check(cfg.KubeClientConfig.Complete())
+		},
+		Run: func(_ *cobra.Command, _ []string) {
+			clierror.Check(runConfig(&cfg))
+		},
+	}
+
+	cfg.KubeClientConfig.AddFlag(cmd)
+	cmd.Flags().BoolVar(&cfg.dockerconfig, "dockerconfig", false, "Generate a docker config.json file for the Kyma registry")
+	cmd.Flags().BoolVar(&cfg.externalurl, "externalurl", false, "External URL for the Kyma registry")
```

Comment:
I would suggest replacing the `--externalurl` flag with the `--external-url` one.

Also, the description is confusing because it does not describe what is causing this flag. Maybe this would be better:

```
Print docker registry external URL
```

Reply:
In general, I don't get the goal of this flag, because it's not cooperating with the `--output` flag and exactly the same information will be located in the `config.json` after running the `registry config --dockerconfig` command. Maybe we should remove this flag? or move it to a command command like `registry external-up`? or I don't know...

But this is only a suggestion. For sure this flag should be re-designed

Reply:
Done, can't do both at once now